AstronomyTests/MockLoader.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  MockLoader.swift
+//  AstronomyTests
+//
+//  Created by Cora Jacobson on 9/30/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import Foundation
+@testable import Astronomy
+
+class MockLoader: NetworkDataLoader {
+
+    var data: Data?
+    var error: Error?
+    
+    func loadData(from request: URLRequest, completion: @escaping (Data?, Error?) -> Void) {
+        DispatchQueue.main.asyncAfter(deadline: .now() + 1) {
```

Comment:
Adding `1` to `.now()` makes this wait 1 whole second. Not a huge deal with just a few tests, but if the app was large and had a ton of tests that got run multiple times by every developer each day... it adds up. I would say something on the order of milliseconds should be sufficient.